### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by doogstar**
#1853 Creating Forms 
Checked in V8 and updated as appropriate

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for running through this @doogstar 😄 

Seems you've removed the Umbraco 7 version entirely - we still want to keep that one!
Could you create a new Umbraco 7 index files for this? It should be called `index-v7.md` with a YAML tag that says `versionFrom: 7.0.0` ? 😁

**Comment by sofietoft** [comment]
Ah, whoops! Didn't realize you'd made another PR as well 🙈 
My bad!

**Comment by sofietoft** [comment]
Related: #1996

File: Getting-Started/Code/Creating-Forms/index.md [hunk 1]
```diff
@@ -104,7 +103,7 @@ You can add the form to a template by rendering the partial view:
 To add the form to your site we'll make a macro. This also makes it possible to let editors add the form to a page using the rich text editor.
 
 #### Creating a macro
-Go to the developer section and right click the menu icon on the __Partial Views Macro Files__ node. Name the macro *Contact Form*, select an empty snippet and leave the __Create Macro__ checkbox checked.
+Go to settings and click the menu icon on the __Partial Views Macro Files__ node and select __New partial view macro__. Name the macro *Contact Form*.
```
Comment (sofietoft) [inline Getting-Started/Code/Creating-Forms/index.md]:
```suggestion
Go to the Settings section and right-click the __Partial Views Macro Files__ node. Choose "Create" and select __New partial view macro__. Name the macro *Contact Form*.
```

**Comment by doogstar** [comment]
> 
> 
> I've finally had some time to review here - it looks great! 😄
> 
> I've added a single suggestion. Could you take a look?

Thanks, makes sense :)

**Comment by sofietoft** [comment]
I've given you the **C-Trib** badge 😎 

Thanks again!
I look forward to seeing you back here with more excellent contributions 😁

**Pull request merged**